Client/Application/AppDelegate.swift

```diff
@@ -243,6 +243,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
             log.error("Failed to initialize user referral program")
             UrpLog.log("Failed to initialize user referral program")
         }
+        
+        if !Preferences.Migration.syncOrderCompleted.value {
```

Comment:
I think we should consider doing the following.

Adding a static function on `Preferences.Migration` called `launchMigrations` or something that we just call. It will then run through each necessary migration and handle the flags internally. I do not want to continue adding tons of `Preferences.Migration.foo` to `AppDelegate`, as we will just continue having more and more of these.

Then when a new migration is necessary, we just internally add it to the `Preferences.Migration` function.

This also let's all migration preferences to be internalized. We can leave the existing migration call where it is, but I think this (and future ones where possible), should be moved internally.